src/ophys_etl/workflows/on_prem/dags/ophys_processing.py

```diff
@@ -223,15 +226,33 @@ def run_inference():
         thumbnail_dir >> run_inference()
 
     @task_group
-    def trace_extraction(motion_corrected_ophys_movie_file):
-        run_workflow_step(
+    def trace_extraction(motion_corrected_ophys_movie_file,
+                         rois_file):
+        module_outputs = run_workflow_step(
             slurm_config_filename="trace_extraction.yml",
             module=TraceExtractionModule,
             workflow_step_name=WorkflowStepEnum.TRACE_EXTRACTION,
             workflow_name=WORKFLOW_NAME,
             docker_tag=app_config.pipeline_steps.trace_extraction.docker_tag,
             module_kwargs={
-                "motion_corrected_ophys_movie_file": motion_corrected_ophys_movie_file # noqa E501
+                "motion_corrected_ophys_movie_file": motion_corrected_ophys_movie_file, # noqa E501
+                "rois_file": rois_file
+            },
+        )
+
+        return module_outputs[WellKnownFileTypeEnum.ROI_TRACE.value]
+
+    @task_group
+    def demix_traces(motion_corrected_ophys_movie_file, roi_traces_file):
+        run_workflow_step(
+            slurm_config_filename="demix_traces.yml",
```

Comment:
This file needs to be added

Reply:
you can also leave this argument out and it will use the settings in `default.yml`